sys/posix/pthread/pthread.c

```diff
@@ -206,7 +206,7 @@ void pthread_exit(void *retval)
             }
         }
 
-        dINT();
```

Comment:
Yes, this assumption is correct, (i.e. it was correct when this code was written).